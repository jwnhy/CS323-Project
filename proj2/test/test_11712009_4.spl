int main(int a, int b) {
    int f = a(b);
    int f = 1;
}