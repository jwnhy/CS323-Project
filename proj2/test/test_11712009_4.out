Error type 2 at Line 2: Function not found.
Error type 5 at Line 2: Two sides of assignment have inconsistent type.
Error type 3 at Line 3: Variable Redefinition.
